.github/workflows/label.yml

```diff
@@ -10,6 +10,8 @@ on:
       - opened
       - convert_to_draft
       - ready_for_review
+      - labeled
```

Comment:
@zhawkins I'm not sure about this one - it might cause an infinite loop.  I need to check.